modules/persistent-storage-hostpath-pod.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-hostpath.adoc
+
+[id="persistent-storage-hostpath-pod_{context}"]
+= Mounting the hostPath share in a privileged Pod
+
+After the PersistentVolumeClaim has been created, it can be used inside by an application. The following example demonstrates mounting this share inside of a Pod.
+
+.Prerequisites
+* A PersistentVolumeClaim exists that is mapped to the underlying hostPath share.
+
+.Procedure
+
+* Create a privileged Pod that mounts the existing PersistentVolumeClaim:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-name <1>
+spec:
+  containers:
+    ...
+    securityContext:
+      privileged: true <2>
+    volumeMounts:
+    - mountPath: /data <3>
+      name: hostpath-privileged
+  ...
+  securityContext: {}
+  volumes:
+    - name: hostpath-privileged
+      persistentVolumeClaim:
+        claimName: task-pvc-volume <4>
+----
+<1> The name of the Pod.
+<2> The Pod must run as privileged to grant access from mounted storage to data across Pods in the same node.
```

Comment:
This sentence is a bit wordy. I think we can just say `The Pod must run as privileged to access the node's storage.`